crates/bevy_text/src/text.rs

```diff
@@ -127,7 +52,7 @@ impl Text {
 }
 
 /// Contains the value of the text in a section and how it should be styled.
-#[derive(Debug, Default, Clone, Reflect)]
+#[derive(Component, Debug, Default, Clone, Reflect)]
 #[reflect(Default)]
 pub struct TextSection {
```

Comment:
This should be split up I think, `TextStyle` needs to be a separate elidable component imo.

Reply:
Not sure how I feel about that, it would require a LOT more changes, that I don't want to update 100 examples for again.

Reply:
I know but I think it's really necessary.
Maybe if `TextSection` is made into a bundle with a `Text` and `TextStyle` components it would just work without any changes to most of the examples?

Reply:
honestly great idea!

Reply:
> I know but I think it's really necessary.

Could you elaborate on this?

Reply:
I think splitting up into `Text` and `TextStyle` components is a good idea, but what should the bundle be called? Just `TextSection`, or `TextSectionBundle`? what do you think?

Reply:
That sounds good to me, but it also sounds like something that should be discussed/designed. Would be it acceptable to catalog this in an issue and implement it in a followup PR?

Reply:
Totally agree on splitting this, strongly feel this should be moved to followup.

Reply:
Yep I've no problem with moving this to a followup, it would be good if we can avoid multiple rewrites of the api and examples though.

Reply:
It probably should be split up more even, previously it's really bugged me how changing the colour of a single section causes a relayout of the whole text node. Separate `TextColor` component disappears that problem.